client/extensions/woocommerce/lib/order-values/test/fixtures/order-with-refunds.js

```diff
@@ -0,0 +1,80 @@
+/** @format */
+
+export default {
+	line_items: [
+		{
+			id: 15,
+			name: 'Scarf - Striped',
+			product_id: 49,
+			taxes: [
+				{
+					id: 4,
+					total: '5.3964',
+					subtotal: '6.3487',
```

Comment:
I guess this doesn't matter much since this is a test, but under what circumstances would the subtotal exceed the total? (I'm a big believer in realistic test data.)

Also, how did you decide to present taxes and prices to 4 decimal points?

Reply:
There's a coupon applied on this test data, so the subtotal is the product cost, and total is minus the coupon. This data came from my test site, that decision was made by WC core 🙂